source/hardware/controllers/akai_mpk_mini_2.rst

```diff
@@ -0,0 +1,23 @@
+AKAI MPK Mini mkII
+===================
+
+The *AKAI MPK Mini mkII* is a funny keyboard controller for musicians.
+This device is not really conceived for DJing but this adaptation can be funny.
```

Comment:
```suggestion
This device is not really conceived for DJing but could be useful for radio broadcasting setups. 
```

?

Reply:
Could be, I didn't think about radio broadcasting